docs/modules/configuration/pages/reporting.adoc

```diff
@@ -178,6 +178,39 @@ To change the brand title in report use the property `report.brand.title=<title>
 report.brand.title=MY BRAND
 ----
 
+== Configure defect categories
+
+It is possible to customize the names of defect categories in the report, as well as the statuses of tests that will be categorized under these names. +
+Use the next properties:
+
+[source,properties]
+----
+report.result.categories.<category-identifier>.name=<name>
+report.result.categories.<category-identifier>.statuses=<statuses>
+----
+
+* `category-identifier` - The unique identifier of category.
+* `name` - The desired category name.
+* `statuses` - The comma-separated list of test statuses included in this category. Possible values:
+https://allurereport.org/docs/test-statuses/#failed[failed],
+https://allurereport.org/docs/test-statuses/#broken[broken],
+unknown (in context of VIVIDUS - test cases with xref:tests-configuration.adoc#_known_issues[Known issues]),
+https://allurereport.org/docs/test-statuses/#skipped[skipped].
```

Comment:
is it possible to map passed tests as well?

Reply:
> is it possible to map passed tests as well?

Yep, but I didn't include it in the doc because not quite sure that it is correct to include Passed tests in this context (defect categories)

Reply:
just use `categories` instead of `defect categories`